### ISSUE TITLE
Confusing account balances

### ISSUE THREAD

**Issue by JustinBeBoy** [crypto-power/cryptopower#577]
I have 3 accounts: default, mixed11, Unmixed like screenshot bellow, I have DCR on Unmixed:
<img width="810" alt="b1" src="https://github.com/user-attachments/assets/0ceb78d5-4688-42b7-a78c-51959e6376da">

I set mixed is Unmixed account and unmixed is default account in stakshuffle page:
<img width="820" alt="b2" src="https://github.com/user-attachments/assets/63a92d8e-8e89-4db4-92ba-a3e3dd117120">

after that, I sent some dcr from Unmixed account like bellow:
<img width="871" alt="b4" src="https://github.com/user-attachments/assets/7183f3fa-84cc-4ff9-9b3b-f18437d43059">

after sent, all DCR balance in Unmixed move to default account and stakshuffle page show like bellow
<img width="811" alt="b6" src="https://github.com/user-attachments/assets/4aa9ba77-2ef6-41b8-bb4d-4e32af47931b">
<img width="846" alt="b7" src="https://github.com/user-attachments/assets/33d5be37-b4d8-4609-b8e4-a0188101aec5">

@dreacot Is it a bug ?

**Comment by dreacot** [comment]
i don't think so since, the balances now reflect what you set them to, i.e sending from mixed (unmixed) to unmixed (defualt)

**Comment by JustinBeBoy** [comment]
> i don't think so since, the balances now reflect what you set them to, i.e sending from mixed (unmixed) to unmixed (defualt)

I talking about why can balance move between 2 accounts while mixer is not running, i did not transfer money between those 2 accounts

**Comment by JustinBeBoy** [comment]
> Do you have the txid? It looks like the change address was taken from default and it spent one UTXO sending all the change as well back to default. What behavior was expected? Dealing with multiple accounts is kind of confusing.

it seems logical that after sending an amount, uxto will return the money to the mixed wallet, but would that make it confusing for users?
